blockly-core/core/ui/block.js

```diff
@@ -17,6 +17,8 @@
  * limitations under the License.
  */
 
+/* global Blockly, goog */
```

Comment:
I didn't even realize we were running eslint against blockly-core.  Seems like it should have its own linting solution.